xyo-android-library/src/main/java/network/xyo/sdk/XyoBleClient.kt

```diff
@@ -30,20 +30,20 @@ class XyoBleClient(
     scan: Boolean
 ) : XyoClient(relayNode, procedureCatalog, autoBoundWitness) {
 
-    override var autoBridge: Boolean = false
-    override var acceptBridging: Boolean = false
+    override var autoBridge: Boolean = true
```

Comment:
I'm not sure if changing the boolean really affects bound witness performance, unless you simply want to set up the environment without user input.

Reply:
It doesn't affect, the main intention is really to setup the environment without user input.